js/src/features/project/team.reducer.ts

```diff
@@ -2,19 +2,25 @@ import { createReducer } from "typesafe-actions";
 
 import { ProjectTeam } from "ProjectModels";
 import { FormErrorMap } from "MyTypes";
-import { fetchProjectTeamAsync } from "./actions";
+import {
+  closeProjectTeamAddMemberFormAction,
+  fetchProjectTeamAsync,
+  openProjectTeamAddMemberFormAction,
+} from "./actions";
 import { normalizeErrors } from "../../utils/error";
 
-export interface State {
+export type State = {
   data: ProjectTeam | null;
   loading: boolean;
   errors: FormErrorMap;
+  isOpenAddMemberForm: boolean;
```

Comment:
This prop looks a little bit ugly, because one variable contains 2 verbs. I suggest to use `isOpenNewMemberForm`

Reply:
And you can use it on all the same cases

Reply:
Fixed